src/tree/sender_ratchet.rs

```diff
@@ -79,16 +79,16 @@ impl SenderRatchet {
                 self.past_secrets.push(new_secret);
             }
             let secret = self.past_secrets.last().unwrap();
-            let application_secrets = self.derive_key_nonce(&secret, generation, ciphersuite);
+            let ratchet_secrets = self.derive_key_nonce(&secret, generation, ciphersuite);
             self.generation = generation;
-            Ok(application_secrets)
+            Ok(ratchet_secrets)
         }
     }
     fn ratchet_secret(&self, secret: &[u8], ciphersuite: &Ciphersuite) -> Vec<u8> {
```

Comment:
If the result is a `secret`, it should probably not be `Vec<u8>`, but some struct encapsulating the scret, right? Not sure where this is mean to be used, as it isn't `pub` and it I can't see it being used anywhere here.

Reply:
Will be addressed in #94